scripts/eslint/es6Config.js

```diff
@@ -0,0 +1,16 @@
+const eslintrc = require('../../.eslintrc');
```

Comment:
I don't see how this could possibly work. By the pattern, this file should be treated as ES5. But then `const` should not parse. However it lints just fine.

This brings up a separate point: we're actually okay with treating Node scripts as ES6. It's just that we don't want trailing commas in them. So maybe we need three different file types (ESnext+JSX, ES6, and ES5).

Reply:
So the main different lint setting of ES6 and ESnext is we turn off `comma-dangle` rule at es6?

Reply:
JSX, Flow should also be disabled in ES6. And it should use the espree parser.

Reply:
OK, so there are some files in `scripts` should be ignored.

Like: 
[/scripts/flow/environment.js](https://github.com/facebook/react/blob/master/scripts/flow/environment.js)
[/scripts/rollup/shims/react-native/NativeMethodsMixin.js](https://github.com/facebook/react/blob/master/scripts/rollup/shims/react-native/NativeMethodsMixin.js)

Reply:
All shims should be treated as source (ESNext). Good catch.